integration/common/openlineage/common/provider/great_expectations/action.py

```diff
@@ -36,8 +37,9 @@ class OpenLineageValidationAction(ValidationAction):
 
     Openlineage host parameters can be passed in as constructor arguments or environment variables
     will be searched. Job information can optionally be passed in as constructor arguments or the
-    great expectations suite name and batch identifier will be used as the job name
-    (the namespace should be passed in as either a constructor arg or as an environment variable).
+    great expectations suite name and either run_name or batch identifier will be used as the job
+    name - the namespace should be passed in as either a constructor arg or as an environment
+    variable.
```

Comment:
I would provide examples for GE the `suite name`, `run_name` and `batch identifier` (and how they are used to build the job name). I'd also document the environment variable for setting the namespace.

Reply:
Removed that part of PR.